src/System.Runtime/tests/System/StringTests.cs

```diff
@@ -828,7 +828,7 @@ public static void EndsWith(string s, string value, StringComparison comparisonT
         [InlineData("abcdefghijklmnopqrstuvwxyz", 'z', true)]
         public static void EndsWith(string s, char value, bool expected)
         {
-            Assert.Equal(expected, s.EndsWith(value));
+            Assert.Equal(expected, s.EndsWith(value.ToString()));
```

Comment:
The `.ToString()` should be reverted and the test should be annotated with `[SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]`.